queries.sql

```diff
@@ -86,4 +86,64 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals_owned
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
-ORDER BY num_animals_owned DESC;
\ No newline at end of file
+ORDER BY num_animals_owned DESC;
+
+SELECT animals.name 
+FROM animals 
+JOIN visits ON animals.id = visits.animal_id 
+JOIN vets ON vets.id = visits.vet_id 
+WHERE vets.name = 'William Tatcher' 
+ORDER BY visits.visit_date DESC 
+LIMIT 1;
+
+SELECT COUNT(DISTINCT animal_id) FROM visits 
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+SELECT vets.name,specializations.species_id AS specialization_species_id FROM vets
+LEFT JOIN specializations ON vets.id = specializations.vet_id
+ORDER BY vets.name;
```

Comment:
- [ Required ] I think this query will not give you the _List all vets and their specialties, including vets with no specialties_, to do that you need to create joins between `vets`, `specializations`, and `species` tables. You can do this by refactoring your query as shown below.

```ruby
  SELECT vets.*, species.name
  FROM vets
  LEFT JOIN specializations ON vets.id = specializations.vet_id
  LEFT JOIN species ON specializations.species_id = species.id;
```

Reply:
@Felix45 This query returns exactly   List all vets and their specialties, including vets with no specialties as in the screenshot   the * wildcard adds unnecessary information about the vets